installation/scripts/minikube.ps1

```diff
@@ -15,7 +15,7 @@ Write-Output @"
 "@
 
 function CheckIfMinikubeIsInitialized() {
-    $cmd = "minikube status --format '{{.MinikubeStatus}}'"
+    $cmd = "minikube status --format '{{.Host}}'"

```

Comment:
This change also needs to be done in line 70

Reply:
Thanks.